src/webgpu/shader/execution/expression/call/builtin/subgroupAdd.spec.ts

```diff
@@ -269,6 +276,10 @@ function checkPredicatedAddition(
           expected += j;
         }
       }
+
+      if (operation === 'subgroupInclusiveAdd') {
```

Comment:
This feels like it would be better placed as part of the bounds calculations at line 273.